source/appModules/explorer.py

```diff
@@ -174,7 +174,10 @@ class UIProperty(UIA):
 	
 	def _get_value(self):
 		value = super(UIProperty, self).value
-		return value.replace(CHAR_LTR_MARK,'').replace(CHAR_RTL_MARK,'')
+		if value == None:
+			return value
+		else:

```

Comment:
My personal style would be to remove this `else` statement and just have the two return statements on consecutive lines, with the first one indented of course.